backend/tests/api/endpoints/test_remote_devices_api.py

```
@@ -0,0 +1,117 @@
+        db=test_db,
+        current_user=test_user,
+    )
+
+    device = (
+        test_db.query(Kind)
+        .filter(
+            Kind.user_id == test_user.id,
+            Kind.kind == "Device",
+            Kind.namespace == "default",
+            Kind.name == response.device_id,
+            Kind.is_active == True,
```

Ruff (0.15.17)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)